### PR TITLE
replace defer.returnValue with `return`

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Review by adiroiban — APPROVED** [review]
I don't understand why @inlineCallbacks now works with a normal return.

The changes look ok.
I think that if @wsanchez doesn't have anything against this change, this can be merged.

**Comment by graingert** [comment]
I've got 13 errors left to fix on https://github.com/twisted/klein/pull/476

Once that's in, this pr will be trivial to regenerate on top

**Comment by wsanchez** [comment]
@graingert all set

File: src/klein/_session.py [hunk 1]
```diff
@@ -176,8 +176,8 @@ def procureSession(
             )
         if sentSecurely or not request.isSecure():
             # Do not cache the insecure session on the secure request, thanks.
-            cast(Componentized, request).setComponent(ISession, session)
-        returnValue(session)
+            request.setComponent(ISession, session)
```
Comment (graingert) [inline src/klein/_session.py:179]:
```suggestion
            cast(Componentized, request).setComponent(ISession, session)
```

**Pull request merged**